File: src/model_v4/entities/SBS_AUSSOND.ts
import { Column, Entity, Index, PrimaryColumn } from "typeorm";


@Entity("SBS_AUSSOND",{schema:"sbsdb"})
@Index("sbsaussond_index2",["AUSS_DAT",])
@Index("sbsaussond_index1",["SER_NR",])
export class SBS_AUSSOND {

    @PrimaryColumn("bigint",{ 
        nullable:false,
        primary:true,
        name:"HW_INDEX"
        })
    HW_INDEX:string;
        

    @Column("date",{ 
        nullable:true,
        default: () => "'NULL'",
        name:"ANSCH_DAT"
        })
    ANSCH_DAT:string | null;
        

    @Column("double",{ 
        nullable:true,
        default: () => "'NULL'",
        name:"ANSCH_WERT"
        })
    ANSCH_WERT:number | null;
        

    @Column("date",{ 
        nullable:true,
        default: () => "'NULL'",
        name:"AUSS_DAT"
        })
    AUSS_DAT:string | null;
        

    @Column("varchar",{ 
        nullable:true,
        length:50,
        default: () => "'NULL'",
        name:"AUSS_GRUND"
        })
    AUSS_GRUND:string | null;
        

    @Column("varchar",{ 
        nullable:true,
        length:50,
        default: () => "'NULL'",
        name:"INV_NR"
        })
    INV_NR:string | null;
        

    @Column("bigint",{ 
        nullable:true,
        default: () => "'NULL'",
        name:"KONFIG_INDEX"
        })
    KONFIG_INDEX:string | null;
        

    @Column("varchar",{ 
        nullable:true,
        length:50,
        default: () => "'NULL'",
        name:"MAC"
        })
    MAC:string | null;
        

    @Column("varchar",{ 
        nullable:true,
        length:50,
        default: () => "'NULL'",
        name:"NETBOOTGUID"
        })
    NETBOOTGUID:string | null;
        

    @Column("varchar",{ 
        nullable:false,
        length:50,
        name:"SER_NR"
        })
    SER_NR:string;
        

    @Column("varchar",{ 
        nullable:true,
        length:200,
        default: () => "'NULL'",
        name:"WARTUNG_BEM"
        })
    WARTUNG_BEM:string | null;
        

    @Column("varchar",{ 
        nullable:true,
        length:50,
        default: () => "'NULL'",
        name:"WARTUNG_FA"
        })
    WARTUNG_FA:string | null;
        
}

File: SbsdbServer/sbsdbv4/model/SbsAussond.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace hb.SbsdbServer.sbsdbv4.model {
    [Table("SBS_AUSSOND")]
    public class SbsAussond {
        [Column("HW_INDEX", TypeName = "bigint(20)")]
        public long HwIndex { get; set; }

        [Column("ANSCH_DAT", TypeName = "date")]
        public DateTime? AnschDat { get; set; }

        [Column("ANSCH_WERT")] public double? AnschWert { get; set; }

        [Column("AUSS_DAT", TypeName = "date")]
        public DateTime? AussDat { get; set; }

        [Column("AUSS_GRUND", TypeName = "varchar(50)")]
        public string AussGrund { get; set; }

        [Column("INV_NR", TypeName = "varchar(50)")]
        public string InvNr { get; set; }

        [Column("KONFIG_INDEX", TypeName = "bigint(20)")]
        public long? KonfigIndex { get; set; }

        [Column("MAC", TypeName = "varchar(50)")]
        public string Mac { get; set; }

        [Column("NETBOOTGUID", TypeName = "varchar(50)")]
        public string Netbootguid { get; set; }

        [Required]
        [Column("SER_NR", TypeName = "varchar(50)")]
        public string SerNr { get; set; }

        [Column("WARTUNG_BEM", TypeName = "varchar(200)")]
        public string WartungBem { get; set; }

        [Column("WARTUNG_FA", TypeName = "varchar(50)")]
        public string WartungFa { get; set; }
    }
}
